docs/src/pages/components/tabs/HoverTabs.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import Box from '@material-ui/core/Box';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+
+export default function HoverTabs() {
+  const [value, setValue] = React.useState<false | number>(false);
+
+  return (
+    <Box sx={{ width: '100%', display: 'flex', justifyContent: 'center' }}>
+      <Tabs value={value} onMouseLeave={() => setValue(false)}>
```

Comment:
Are we sure this is a good implementation of the tab pattern? I don't think I would ever want a tab to switch on hover which is already a very problematic UI pattern.


The demo is also not keyboard accessible.

Reply:
Let's remove it from this PR and consider it later, it is not quite related to the fix.